### PR TITLE
Enable tests in test_pytorch_onnx_onnxruntime

### PR DESCRIPTION
**Pull request by KsenijaS**
Enable tests in tests/onnx/test_pytorch_onnx_onnxruntime.py for:
- Einsum
- SoftmaxCrossEntropy 
- NLLLoss
- normalize
- pixel_shuffle
- test_interpolate_no_shape
- test_arange_dynamic
- test_slice_neg_large_negone
 since there is a support in ORT for these operators.

### CODE REVIEW

**Comment by KsenijaS** [comment]
@neginraoof @lara-hdr please review

**Review by houseroad — CHANGES_REQUESTED** [review]
The tests failed

**Comment by KsenijaS** [comment]
> The tests failed

PR https://github.com/pytorch/pytorch/pull/37239 updates ort nightly version, after it's merged I will rebase my branch and the tests should pass.

**Comment by neginraoof** [comment]
@KsenijaS ORT is now updated in master. Please rebase this.
Thanks.

**Review by lara-hdr — APPROVED** [review]
LGTM. thanks!

**Comment by KsenijaS** [comment]
@houseroad CI is green! LGTM.

**Review by houseroad — APPROVED** [review]
Looks good. Thanks!
